src/app/screens/Nostr/Confirm.tsx

```diff
@@ -62,21 +72,29 @@ function NostrConfirm() {
               title={origin.name}
               image={origin.icon}
               url={origin.host}
-              isSmall={false}
+              isSmall={true}
             />
-            <div className="dark:text-white pt-6 mb-4">
-              <p className="mb-2">{t("allow", { host: origin.host })}</p>
-              <p className="dark:text-white">
-                <CheckIcon className="w-5 h-5 mr-2 inline" />
-                {description}
-                {details && (
-                  <>
-                    <br />
-                    <i className="ml-7">{details}</i>
-                  </>
-                )}
-              </p>
+
+            {message && (
+              <ContentMessage
+                heading={tPermissions("nostr.nip04Action", {
+                  host: origin.host,
+                  action: action,
+                  defaultValue: "",
+                })}
+                content={message}
+              />
+            )}
+            <div className="flex justify-center mb-4 gap-4">
+              <Hyperlink onClick={toggleShowDetails}>
+                {showDetails ? t("hide_details") : t("view_details")}
+              </Hyperlink>
             </div>
+            {showDetails && (
+              <div className="whitespace-pre-wrap break-words p-2 mb-4 shadow bg-white rounded-lg dark:bg-surface-02dp text-gray-500 dark:text-gray-400">
+                {tPermissions("nostr.peer")}: {peer}
```

Comment:
is there always a peer in this confirm screen?

Reply:
yup! cause confirm screen is only used for encrypt and decrypt actions. for other nostr actions we have different screens

Reply:
should we then name the screen EncryptDecryptConfirm? or something?

Reply:
resolved